twake/frontend/src/app/scenes/Error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import RouterServices from '../../services/RouterServices';
+import 'app/ui.scss';
+
+type State = {
+  hasError: boolean;
+  error: any;
+};
+
+export default class ErrorBoundary extends React.Component<{}, State> {
+  constructor(props: any) {
```

Comment:
I am not a React dev, but can't we have a real type here?

Reply:
This state can be removed. We don't use this state.

Reply:
(for the error: any state)
For the (props: any), the type is "{}" like in Component<{}, State> @stephanevieira75 
So (props: {})